pkg/generators/openapi.go

```diff
@@ -339,8 +343,18 @@ func (g openAPITypeWriter) generateCall(t *types.Type) error {
 	// Only generate for struct type and ignore the rest
 	switch t.Kind {
 	case types.Struct:
+		if namer.IsPrivateGoName(t.Name.Name) { // skip private types
```

Comment:
Did you encounter one of these types or is this just precautionary?

Reply:
`int64Amount` was leaking out from quantity into `pkg/generated/openapi/zz_generated.openapi.go`